Description: Set the classpath for Groovy (why is this needed?)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -159,6 +159,7 @@
 
 //  To get the details of the "unchecked" issues.
 compileGroovy.options.compilerArgs = ['-Xlint']
+compileGroovy.groovyClasspath = files('/usr/share/java/groovy-all.jar')
 
 [compileGroovy, compileTestGroovy]*.groovyOptions*.fork(memoryInitialSize: '128M', memoryMaximumSize: '512M')
 
@@ -293,6 +294,7 @@
     footer = copyrightString
     include 'groovyx/gpars/**'
     overview = new File('overview.html')
+    groovyClasspath = files('/usr/share/java/groovy-all.jar')
 }
 
 // Only build javadocs as documentation
